packages/client/components/ActivityLibrary/SearchBar.tsx

```diff
@@ -9,18 +9,18 @@ interface Props {
 const SearchBar = (props: Props) => {
   const {searchQuery, onChange} = props
   return (
-    <div className='my-4 ml-2 flex items-center'>
-      <SearchIcon className='text-slate-600' />
+    <div className='ml-2 flex grow items-center rounded-full bg-slate-200 px-4 py-2 outline-none outline-1 focus-within:outline-sky-500'>
```

Comment:
+1 to match the designs, use `outline-offset-0` as now there's a gap between outline and the background